app/src/Cops/Front/Controller/UserBooksController.php

```diff
@@ -139,15 +139,18 @@ protected function displayListAction(Application $app, UserBookCollection $userB
 
         $totalBooks = $books->getRepository()->getTotalRows();
 
+        $addlettersstr = $app['config']->getValue('add_cap_letters');
```

Comment:
Added it by accident :)
This was a rapid development to first get it to work. I want to simplify it now. And also, maybe, solve the mysterious authors sqlite issue. Thanks for noticing!